clients/fossid-webapp/src/main/kotlin/model/status/Description.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.fossid.model.status
+
+interface Description {
```

Comment:
The name `Description` is quite generic. What does it describe? Should the interface be named `ScanDescription`, and another name should be searched for the class? Also some comments would be helpful.

Reply:
So you understand the problem I had: `ScanStatus` is the status of a scan. `Scan` is the scan itself. `ScanState` is the state of a scan. `ScanDescriptionXXX` is the description of a scan in version XXX.
Since here we are talking about a non-version specific scan description, what about `GenericScanDescription` or `UnversionedScanDescription` ?

Reply:
Yes, the terminology used by FossID seems to be quite confusing. Especially the distinction between `ScanStatus` and `ScanState` I find difficult to grasp, but this is out of our control. Regarding `Description`, I would be fine with either name, only the plain `Description` I find rather meaningless.